src-tauri/src/main.rs

```diff
@@ -87,10 +87,10 @@ async fn login(app: tauri::AppHandle, host: &str) -> Result<String, InvokeError>
 
 fn main() {
     tauri::Builder::default()
-        .setup(|app| {
+        .setup(|_app| {
             #[cfg(debug_assertions)] // only include this code on debug builds
             {
-                let window = app.get_window("main").unwrap();
+                let window = _app.get_window("main").unwrap();
```

Comment:
Yeah seems like it's actually meant for that? https://medium.com/codechain/rust-underscore-does-not-bind-fec6a18115a8#:~:text=So%20what%20if%20you%20only,you%20actually%20use%20those%20variables.

I don't remember if I did it because it was suggested by the compiler or nah